src/utils/normalize-mime-parts.ts

```diff
@@ -1,3 +1,12 @@
+/** List of content type not show as attachment */
+const ignoreContentTypeToShowAsAttachment = [
+	'application/pkcs7-signature',
+	'application/x-pkcs7-signature',
+	'message/delivery-status', // present in Undelivered mail
+	'message/disposition-notification', // present in read-receipt response
+	'xml/x-zimbra-share' // present in folder share message
```

Comment:
if I remember correctly, there is ZBUG for this case also, can you find that and link with this ticket?

Reply:
https://jira.corp.synacor.com/browse/ZBUG-2452

Reply:
please get ticket validated with this change